### DISCUSSION TITLE
Manual Face Tagging

### DISCUSSION THREAD

**Discussion by geicorules** [ente-io/ente#3596] [Enhancements]
The AI does a decent job but some times it needs help it would be nice if we could slap a square on the photo over a face and manually tag/ annotate the photo.
The AI seems to be especially weak with animals and it would be nice if I could place some manual tags on faces to help it out.

At this point the only way I can group together photos of my pet ente the duck would be to add text tag # ente onto it but these search parameters are treated differently and these photos wouldn't be grouped into the person ente the duck.

**Comment by laurenspriem** [comment]
The current version of Private AI is not designed to recognize animals specifically. This is something we're planning to add later on, see [this feature request](https://github.com/ente-io/ente/discussions/2167).

**Reply by eric-gitta-moore** [reply]
Hello, may I ask what "Private AI" means? Is that yolov5face?

**Reply by laurenspriem** [reply]
Private AI is the term we plan to use to refer to the collection of all features driven by AI models, so including face detection by YOLOv5Face. We will get more consistent on this naming once the current AI features get out of beta.

**Comment by geicorules** [comment]
Oh that is good to know.
But I have this happen with humans all the time.
I'll have a photo where just 1 or 2 people will be tagged then the next the other 2 or tag and even photos where they are the worst captures of bluey faces yet the AI can recognize a face so not sure why it misses some.

So if we could just click on the faces icon and then tap a missing face that would be great.

On obviously I understand this is much easier said than done.
Thanks
